app/views/IndexView.scala.html

```diff
@@ -27,5 +33,15 @@
 
     <h1 class="govuk-heading-xl">@messages("index.heading")</h1>
 
-    <p class="govuk-body">@messages("index.guidance")</p>
+    <p class="govuk-body">@Html(messages("index.p1", link(controllers.addFinancialInstitution.routes.NameOfFinancialInstitutionController.onPageLoad(mode).url, "index.p1.link")))</p>
```

Comment:
I'd be happy for us to hardcode the "Normal" mode here and remove the need for having a mode parameter on the route, controller and view for this index page.